funcx_endpoint/tests/unit/test_register_endpoint.py

```diff
@@ -99,3 +99,45 @@ def test_register_endpoint_invalid_response(
             endpoint_dir=tmp_path,
             endpoint_name="Some EP Name",
         )
+
+
+@responses.activate
+@pytest.mark.parametrize("multi_tenant", [None, True, False])
+def test_register_endpoint_multi_tenant(
+    tmp_path,
+    endpoint_uuid,
+    other_endpoint_id,
+    register_endpoint_response,
```

Comment:
Doesn't look like `other_endpoint_id` is in use.

Reply:
Removed fixture